.pre-commit-hooks.yaml

```diff
@@ -6,9 +6,11 @@
     types: [python]
 -   id: python-check-mock-methods
     name: check for not-real mock methods
-    description: 'Prevent a common mistake of `assert mck.not_called()` or `assert mck.called_once_with(...)`'
+    description: >
+        Prevent common mistakes of `assert mck.not_called()`, `assert mck.called_once_with(...)` and
+        and `mck.assert_called`.
     language: pygrep
-    entry: 'assert .*\.(not_called|called_)'
+    entry: '(assert .*\.(not_called|called\w*)|\.assert_(not_called|called\w*)($|[^(\w]))'
```

Comment:
why the `\w`?

Also you broke the original regex since `mck.called` is a thing now

Reply:
I want to include the whole variety of `called`, `called_once_with`, etc.

Reply:
Ah, `mck.called`!